application/forms/EditNodeForm.php

```diff
@@ -79,6 +80,13 @@ public function setup()
                 $this->setSubmitLabel($this->translate('Next'));
                 return;
         }
+
+        $label = $this->translate('Delete');
+        $el = $this->createElement('submit', $label, array(
+            'data-base-target' => '_main'
+        ))->setLabel($label)->setDecorators(array('ViewHelper'));
+        $this->deleteButtonName = $el->getName();
+        $this->addElement($el);
```

Comment:
- Please use short array syntax.
- You do not need `$label` variable, add label directly.
- Why not just assign a static name for the delete (`delete_node` for example)?
- Add element directly, without creating it first.
- `deleteButtonName` property is not required.
- Decorator can be added in options array.

Reply:
`deleteButtonName` property needs to be set, for it tob grouped with `submit` button. Check https://github.com/Icinga/icingaweb2-module-businessprocess/blob/cbb6b018da37540fc8655071afea484a4401759c/library/Businessprocess/Web/Form/QuickForm.php#L132.